src/server/stores/user-store.js

```diff
@@ -28,18 +33,27 @@ function validate(password, user) {
   return bcrypt.compareSync(password, user.password);
 }
 
+/**
+ * Gets a user by id
+ * @param {number} id - The id of the user
+ * @returns {Promise.<object|null>} Promise resolving to the user, or null if one cannot be found
+ */
+export function get(id) {
```

Comment:
-Add the "if config.db.isMocked" conditional expression, as done in getByEmail, even if (worst-case scenario) you leave the non-mocked part to me
-Rename the method to getById
-It would have been **really, really easy** to implement this function correctly and connect it to the back-end as done in getByEmail. Let me know if I should do it (in which case I'll extract the common bits to a function as well), but from now on, given that I've laid down the correct foundations, I'd prefer you try to work in accordance with them rather than my sweeping up after you.